core/modules/macros/contrastcolour.js

```diff
@@ -29,6 +29,22 @@ exports.params = [
 Run the macro
 */
 exports.run = function(target,fallbackTarget,colourA,colourB) {
```

Comment:
> But these changes make the contrastcolour macro be specific to tag pills; it's supposed to be a general purpose macro.

@Jermolene it is not specific to tag pills at all. Instead, it defaults to the current tiddlers color value, that's all.